Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/distance.h

```diff
@@ -1310,8 +1322,1295 @@ double approximate_symmetric_Hausdorff_distance(const TriangleMesh& tm1,
+  tm1_parts(s.tm1_parts), tm2(s.tm2),
+  error_bound(s.error_bound), vpm1(s.vpm1), vpm2(s.vpm2),
+  infinity_value(s.infinity_value), initial_bound(s.initial_bound),
+  tm1_trees(s.tm1_trees), tm2_tree(s.tm2_tree), distance(-1.0) {
+    CGAL_assertion(tm1_parts.size() == tm1_trees.size());
+  }
+
+  void operator()(const tbb::blocked_range<std::size_t>& range) {
+    Timer timer;
+    timer.reset();
+    timer.start();
+    double hdist = -1.0;
+    auto stub = CGAL::Emptyset_iterator();
+
+    for (std::size_t i = range.begin(); i != range.end(); ++i) {
+      CGAL_assertion(i < tm1_parts.size());
+      CGAL_assertion(i < tm1_trees.size());
+      const auto& tm1 = tm1_parts[i];
+      const auto& tm1_tree = tm1_trees[i];
+      // TODO: add distance_bound (now it is -FT(1)) in case we use parallel
+      // for checking if two meshes are close.
+      const double dist = bounded_error_Hausdorff_impl<Kernel>(
+        tm1, tm2, error_bound, vpm1, vpm2,
+        infinity_value, initial_bound, -FT(1),
+        tm1_tree, tm2_tree, stub);
+      if (dist > hdist) hdist = dist;
+    }
+    if (hdist > distance) distance = hdist;
+    timer.stop();
+    // std::cout << "* time operator() computation (sec.): " << timer.time() << std::endl;
+  }
+
+  void join(Bounded_error_distance_computation& rhs) {
+    distance = (CGAL::max)(rhs.distance, distance);
+  }
+};
+
+#endif // defined(CGAL_LINKED_WITH_TBB) && defined(CGAL_METIS_ENABLED)
+
+template< class Concurrency_tag,
+          class Kernel,
+          class TriangleMesh1,
+          class TriangleMesh2,
+          class VPM1,
+          class VPM2,
+          class NamedParameters1,
+          class NamedParameters2,
+          class OutputIterator >
+double bounded_error_one_sided_Hausdorff_impl(
+  const TriangleMesh1& tm1,
+  const TriangleMesh2& tm2,
+  const typename Kernel::FT error_bound,
+  const typename Kernel::FT distance_bound,
+  const bool compare_meshes,
+  const VPM1& vpm1,
+  const VPM2& vpm2,
+  const NamedParameters1& np1,
+  const NamedParameters2& np2,
+  OutputIterator& out)
+{
+  #if !defined(CGAL_LINKED_WITH_TBB) || !defined(CGAL_METIS_ENABLED)
+  CGAL_static_assertion_msg(
+    !(boost::is_convertible<Concurrency_tag, CGAL::Parallel_tag>::value),
+    "Parallel_tag is enabled but at least TBB or METIS is unavailable.");
+  #endif
+
+  using FT = typename Kernel::FT;
+
+  using TM1 = TriangleMesh1;
+  using TM2 = TriangleMesh2;
+
+  using TM1_primitive = AABB_face_graph_triangle_primitive<TM1, VPM1>;
+  using TM2_primitive = AABB_face_graph_triangle_primitive<TM2, VPM2>;
+
+  using TM1_traits = AABB_traits<Kernel, TM1_primitive>;
+  using TM2_traits = AABB_traits<Kernel, TM2_primitive>;
+
+  using TM1_tree = AABB_tree<TM1_traits>;
+  using TM2_tree = AABB_tree<TM2_traits>;
+
+  using Face_handle_1 = typename boost::graph_traits<TM1>::face_descriptor;
+  using Face_handle_2 = typename boost::graph_traits<TM2>::face_descriptor;
+
+  using Timer = CGAL::Real_timer;
+
```

Comment:
add a comment why it is not ON by default

Reply:
Done.